.github/workflows/loadtest.yml

```diff
@@ -0,0 +1,137 @@
+---
+  name: Load Test
+  on:  # yamllint disable-line rule:truthy
+    workflow_dispatch:
+      inputs:
+        runner:
+          description: The runner to execute on
+          default: 'ubuntu-latest'
+          type: choice
+          options:
+            - ubuntu-latest
+            - devnet
+            - testnet
+        environment:
+          description: The environment to run against
+          required: false
+          type: environment
+        scenario:
+          default: 'simple'
+          description: The scenario to run
+          type: string
+    workflow_call:
+      inputs:
+        environment:
+          description: The environment to run against
+          type: string
+          required: true
+        runner:
+          required: true
+          type: string
+          description: The runner label to use
+        scenario:
+          required: true
+          description: The mode for the stress test
+          type: string
+      secrets:
+        SLACK_PERFORMANCE_WEBHOOK_URL:
+          required: true
+        DD_API_KEY:
+          required: true
+        RPC_URL:
+          required: true
+        LOADTEST_MNEMONIC:
+          required: true
+
+  
+  jobs:
+    run_k6:
+      environment: ${{ inputs.environment }}
+      runs-on: ubuntu-latest
+      steps:
+        - name: Install Go
+          uses: actions/setup-go@v3
+          with:
+            go-version: 1.18.x
+  
+        - name: Checkout code
+          uses: actions/checkout@v3
+        
+        - uses: datadog/agent-github-action@v1.3
+          with:
+            api_key: ${{ secrets.DD_API_KEY }}
+        
+        - name: Install xk6
+          run: |
+            go install go.k6.io/xk6/cmd/xk6@latest
+            xk6 build --with github.com/distribworks/xk6-ethereum@a3127e8173097e725da2890ee61782b1bea13fb7
```

Comment:
Have you considered creating a release for xk6?

Reply:
That is Blockspeed https://github.com/distribworks/blockspeed/releases